src/resources/bucket.ts

```diff
@@ -55,6 +57,15 @@ class BucketResource extends Base<BucketPermission> {
         }
       );
     });
+
+    this.resourcePromise = new Promise<Resource>((res, rej) => {
```

Comment:
This isn't required as the resourcePromise is set in the make method

Reply:
fixed